### ISSUE TITLE
Links with curlywurlies break

### ISSUE THREAD

**Issue by neoel** [pngwn/MDsveX#31]
When composing a link with a curlywurly inside it fails to work. When using a solo curlywurly it does work.
```
` ` `js exec
   let variable = 'testing'
` ` `
> [Compound link](/base/{variable})
```

**Comment by pngwn** [comment]
That is very interesting. I'm crazy busy right now but as soon as next week is over I'll be back working on the rewrite of MDsveX and I'll sort this out.

**Comment by pngwn** [comment]
This is fixed in 0.8. I have tests.
